packages/pmml-editor/src/editor/MockHistoryUI.tsx

```diff
@@ -51,14 +56,23 @@ const StateButtons = () => {
 
 const HistoryLog = () => {
   const [state, setState] = useState(true);
+  const service: HistoryService = useContext(HistoryContext).service;
```

Comment:
```suggestion
  const { service } = useContext(HistoryContext);
```

I think that's more idiomatic :)

Reply:
Changed throughout (I still have yet to learn some of this syntax sugar).